test/dynamo/test_dicts.py

```diff
@@ -163,9 +163,6 @@ def fn(sd, x):
             x = x * sd.get(2, 0)
             x = x * sd.get(3, 4)
             x = len(sd) * x
-            x = x * sd.attr
-            sd.attr = 10
-            x = x * sd.attr
             return x
```

Comment:
Did the removed portion cause a failure?

Reply:
Yes, it uncovered cases where there is mutation on the dict but no attribute mutation. Earlier, this was missed. We have other test that test for attr mutation, so we are covered in general.